### DISCUSSION TITLE
Development Emulator Support

### DISCUSSION THREAD

**Discussion by tertty** [joeycastillo/Sensor-Watch#34] [General]
Hello, and congrats on reaching your funding goal so quickly! 

Just curious if there were any plans for a Sensor Watch emulator to jumpstart early development before the boards get shipped out. I figured this would also be helpful for future development in general. Thanks!

**Comment by joeycastillo** [comment]
Thanks so much! 

In re: your question, things are happening on that front, and I sense we may have some news on a Sensor Watch emulator in the coming days. [More details here!](https://github.com/joeycastillo/Sensor-Watch/pull/35)

**Reply by tertty** [reply]
Yes I actually saw this PR earlier today. I'll be waiting for the news, super exciting!
